extensions/amp-list/0.1/amp-list.js

```diff
@@ -102,7 +96,11 @@ export class AmpList extends AMP.BaseElement {
     if (src !== undefined) {
       const typeOfSrc = typeof src;
       if (typeOfSrc === 'string') {
-        this.fetchList_();
+        // Don't fetch if we aren't laid out yet since we also fetch in
+        // layoutCallback().
+        if (this.getLayoutWidth() > 0) {
```

Comment:
for later: Might be worth adding a new values to `element.readyState` that mimic the signals (e.g. `load_start`, `load_end`) etc.. rather than observing the effect of layoutcallback.